pkg/cli/debug/debug.go

```diff
@@ -99,7 +99,13 @@ var (
 	  %[1]s daemonset/test -c second -- /bin/env
 
 	  # See the pod that would be created to debug
-	  %[1]s mypod-9xbc -o yaml`)
+	  %[1]s mypod-9xbc -o yaml
+
+	  # Debug a resource but launch the debug pod in another namespace.
+	  # Note: Not all resources can be debugged using --to-namespace without modification. For example,
+	  # volumes and serviceaccounts are namespace-dependent. Add '-o yaml' to output the debug pod definition
+	  # to disk.  If necessary, edit the definition then run 'oc debug -f -' or run without --to-namespace.
+	  %[1]s node/node-m-0.openshift.example.internal --to-namespace testns`)
```

Comment:
User simpler example `%[1]s mypod-9xbc --to-namespace testns`, debugging node is a heavy gun :wink:

Reply:
ok, i updated to use pod instead.